src/pages/components/Demo/Demo.tsx

```diff
@@ -76,21 +82,18 @@ const Demo = ({ searchWord, words }) => {
 
   return !isLoading ? (
     <div className="flex justify-center mb-16">
-      <div
-        className="flex flex-col items-center md:items-start xl:flex-row
-        lg:space-x-10 p-4 bg-gradient-to-tl from-blue-100 to-white rounded-md"
-      >
-        <div className="demo-inputs-container space-y-5 bg-white p-4 md:-mt-16 lg:mt-0 shadow-2xl rounded-md mb-8">
+      <div className="flex flex-col items-center p-4 rounded-md md:items-start xl:flex-row lg:space-x-10 bg-gradient-to-tl from-blue-100 to-white">
+        <div className="p-4 mb-8 space-y-5 bg-white rounded-md shadow-2xl demo-inputs-container md:-mt-16 lg:mt-0">
           <form onSubmit={onSubmit} className="flex flex-col w-full space-y-5">
             <h2>{t('Enter a word below')}</h2>
             <p className="self-center md:self-start">
               {t("Enter a word in either English or Igbo to see it's information")}
             </p>
             <Input
               size="large"
-              onInput={(e) => setKeyword(e.target.value)}
+              onInput={(e: React.ChangeEvent<HTMLInputElement>) => setKeyword(e.target.value)}
               onKeyPress={onEnter}
-              className="h-12 w-full border-gray-600 border-solid border-2 rounded-md px-3 py-5"
+              className="w-full h-12 px-3 py-5 border-2 border-gray-600 border-solid rounded-md"
```

Comment:
again, i don't think we should be changing the order of className values (unless this is automatically done by prettier)

Reply:
This is automatically done by prettier